config/software/ruby-cleanup.rb

```diff
@@ -59,6 +59,17 @@
   delete "#{install_dir}/embedded/share/info"
   delete "#{install_dir}/embedded/info"
 
+  block "Remove leftovers from compiling gems" do
+    # find the embedded ruby gems dir and clean it up for globbing
+    target_dir = "#{install_dir}/embedded/lib/ruby/gems/*/".tr('\\', "/")
+
+    # find gem_make.out and mkmf.log files
+    Dir.glob(Dir.glob("#{target_dir}/**/{gem_make.out,mkmf.log}")).each do |f|
```

Comment:
We could probably include the generated `Makefile`s, too.

Reply:
where are those? This dir looks pretty clean now